ui/preference/list_preference.go

```diff
@@ -44,17 +48,23 @@ func NewListPreference(wallet *wallet.Wallet, theme *decredmaterial.Theme, prefe
 		wallet:        wallet,
 		preferenceKey: preferenceKey,
 		defaultValue:  defaultValue,
-		theme:         theme,
+		theme:         l.Theme,
```

Comment:
```suggestion
		Load:         l,
```

Reply:
why to store the Load if i just need the theme ?